rebalancer.py

```diff
@@ -169,18 +173,40 @@ def auto_enable():
         apdays = int(LocalSettings.objects.filter(key='AR-APDays')[0].value)
     else:
         LocalSettings(key='AR-APDays', value='7').save()
-        apdays = 7        
+        apdays = 7
     if enabled == 1:
-        channels = Channels.objects.filter(is_active=True, is_open=True, private=False).annotate(outbound_percent=((Sum('local_balance')+Sum('pending_outbound'))*1000)/Sum('capacity')).annotate(inbound_percent=((Sum('remote_balance')+Sum('pending_inbound'))*1000)/Sum('capacity'))
+        channels = Channels.objects.filter(is_active=True, is_open=True, private=False)
```

Comment:
This query may work to produce an aggregated list by remote_pubkey directly, that way we may be able to avoid using the nested loop here.

```
channels = Channels.objects.filter(is_active=True, is_open=True, private=False).values('remote_pubkey').annotate(outbound_percent=((Sum('local_balance')+Sum('pending_outbound'))*1000)/Sum('capacity')).annotate(inbound_percent=((Sum('remote_balance')+Sum('pending_inbound'))*1000)/Sum('capacity').order_by())
```

Reply:
Thank you,
Won't we still need the nested loop for calculating iapD and oapD for the aggregated peer level instead of individual channel level? In that case, can I keep the nested loop only for iapD and oadD?

Reply:
I think I ended up creating PR #94 - can we move this discussion there?